read-the-docs/source/Bugzilla.rst

```diff
@@ -30,11 +30,14 @@ get_ticket_content()
 
 ``get_ticket_content(self, ticket_id=None)``
 
-Queries the Bugzilla API to get ticket_content using ticket_id.
+Queries the Bugzilla API to get ticket_content using ticket_id. The
+ticket_content is expressed in a form of dictionary as a result of JIRA API
```

Comment:
Oops, there's a reference to JIRA here - it should be Bugzilla.

Reply:
Oh, s***. Fixed.